exonum-java-binding/core/rust/src/proxy/node.rs

```diff
@@ -124,10 +112,17 @@ pub extern "system" fn Java_com_exonum_binding_core_service_NodeProxy_nativeSubm
             &env,
             || -> JniResult<jbyteArray> {
                 let payload = env.convert_byte_array(payload)?;
-                let service_transaction =
-                    ServiceTransaction::from_raw_unchecked(transaction_id as u16, payload);
-                let raw_transaction = RawTransaction::new(service_id as u16, service_transaction);
-                match node.submit(raw_transaction) {
+                let tx = match Verified::from_bytes(payload.into()) {
```

Comment:
This code cannot be correct because [Verified::from_bytes](https://github.com/exonum/exonum/blob/b542dae0bc10d8d8c015a347e9ce2c56201cac08/exonum/src/messages/signed.rs#L172-L177) expects a *message*,
but we pass just the payload. This method must create a transaction message and sign it with
the key of this node, ideally, using the core functionality, if it is available.

Second, please move all the logic inside Node#submit and test it. Keep here only
type conversion logic.